README.md

```diff
@@ -1,5 +1,132 @@
+
+  <p align="center">
+    Re:Earth is a state-of-the-art WebGIS platform with expandability through plug-ins and has the ease of running in the browser.
+    <br />
+    <a  href="https://docs.reearth.io/"><strong>Documents >></strong></a>
+    &emsp;&emsp;&emsp;&emsp;
+    <a href="https://docs.reearth.io/oss"><strong>OSS Webpage >></strong></a>
+    <br />
+    <!-- TODO: here -->
+    <a href="">View Demo</a>
+    ·
+    <a href="https://github.com/reearth/reearth/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/reearth/reearth/discussions">Request Feature</a>
+  </p>
+</p>
+
+Re:Earth consist of 2 repositories. Don't forget to check out our [front-end](https://github.com/reearth/reearth-web) and [back-end](https://github.com/reearth/reearth-backend) repositories 👍
+
+
+https://user-images.githubusercontent.com/49897538/123731067-6c802000-d8d2-11eb-9e03-cb7e46eff04e.mp4
+
+
+
+---
+
+<!-- ABOUT THE PROJECT -->
+## Features
+🔌 Highly extensible thanks to the robust plugin system
+
+💻 Super handy being browser-based
+
+💪 Supports standard GIS data formats (CSV, KML, CZML, GeoJSON and shapefile)
+
+📢 Easily make a project public
+
+✨ Freely style the map
+
+
+## Built With
+* [React](https://github.com/facebook/react)
+* [TypeScript](https://github.com/microsoft/TypeScript)
+* [Go](https://github.com/golang/go)
+* [GraphQL](https://github.com/graphql)
+* [MongoDB](https://www.mongodb.com/)
+* [WebAssembly](https://webassembly.org/) + [QuickJS](https://github.com/bellard/quickjs)
+
+
+### Necessities
+
+**Front-end**
+* Node.js: Latest LTS version
+
+**Back-end**
+* Golang +1.16
+* Docker
+
+<!-- USAGE EXAMPLES -->
+## Usage
+
+
+_For examples and more, please refer to the [Documentation](https://docs.reearth.io)_
+
+## Environment
+### OS
+
+| Windows10 | Apple macOS 10.12(macOS Sierra)+ | ChromeOS | iOS 11+ | Android 10+ | Linux(which installed browsers blow) |
+| --------- | --------- | --------- | --------- | --------- | --------- |
+| ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
+
+### Browser
+| ![Edge](https://raw.githubusercontent.com/alrra/browser-logos/master/src/edge/edge_32x32.png) <br />Edge | ![Firefox](https://raw.githubusercontent.com/alrra/browser-logos/master/src/firefox/firefox_32x32.png) <br /> Firefox | ![Chrome](https://raw.githubusercontent.com/alrra/browser-logos/master/src/chrome/chrome_32x32.png) <br /> Chrome | ![Safari](https://raw.githubusercontent.com/alrra/browser-logos/master/src/safari/safari_32x32.png) <br /> Safari | ![iOS Safari](https://raw.githubusercontent.com/alrra/browser-logos/master/src/safari-ios/safari-ios_32x32.png) <br />iOS Safari | ![Chrome for Android](https://raw.githubusercontent.com/alrra/browser-logos/master/src/chrome/chrome_32x32.png) <br/> Chrome for Android |
+| --------- | --------- | --------- | --------- | --------- | --------- |
+| 91+ | 57+| 58+| 11+ | last 2 versions] | last 2 versions
```

Comment:
@KaWaite 
I've added this part. Are `Environment`, `OS`, `Browser` OK? If it's incorrect, I can't have confidence in my English haha.